views/components/hero.njk

```diff
@@ -26,5 +26,15 @@
                 </span>
             {% endif %}
         </div>
+
+        {% if image.caption and  useNewBrand %}
```

Comment:
This code is duplicated from above as the markup order needs to change which is not ideal but we can clean this up fairly swiftly post rebrand.